apps/proxy-auth/src/app/otp/component/login/login.component.html

```diff
@@ -0,0 +1,119 @@
+<div>
+    <proxy-loader *ngIf="isLoading$ | async"></proxy-loader>
+
+    <div *ngIf="step === 1" [formGroup]="loginForm" proxyMarkAllAsTouched>
+        <p>Login</p>
+        <ng-container
+            [ngTemplateOutlet]="formField"
+            [ngTemplateOutletContext]="{
+                label: 'email',
+                formControl: loginForm.get('username')
+            }"
+        >
+        </ng-container>
+        <ng-container
+            [ngTemplateOutlet]="formField"
+            [ngTemplateOutletContext]="{
+                type: 'password',
+                label: 'Password',
+                formControl: loginForm.get('password')
+            }"
+        >
+        </ng-container>
+        <div class="one-line">
+            <div>
+                <a (click)="resetPassword()">Forgot Password?</a>
+            </div>
+            <div class="submit-btn">
+                <button mat-flat-button color="primary" (click)="login()">Login</button>
+            </div>
+        </div>
+    </div>
+
+    <div *ngIf="step === 2" [formGroup]="emailForm" proxyMarkAllAsTouched>
+        <p>Reset Password</p>
+        <ng-container
+            [ngTemplateOutlet]="formField"
+            [ngTemplateOutletContext]="{
+                type: 'email',
+                label: 'Email',
+                formControl: emailForm.get('email'),
+                is_required: true,
+                patternErrorText: 'Name must contain only alphanumeric and underscore'
+            }"
+        >
+        </ng-container>
+        <div>
+            <button mat-flat-button color="primary" (click)="sendOtp()">Send OTP</button>
+        </div>
+    </div>
+
+    <div *ngIf="step === 3" [formGroup]="resetPasswordForm" proxyMarkAllAsTouched>
+        <p>Reset Password</p>
+        <ng-container
+            [ngTemplateOutlet]="formField"
+            [ngTemplateOutletContext]="{
+                type: 'email',
```

Comment:
No Need to show form for email 
show mail as text and added option to change username